BlueShift-iOS-SDK/BlueShiftConfig.m

```diff
@@ -39,10 +43,23 @@ - (BOOL)validateConfigDetails {
     
     if (self.apiKey == NULL || self.apiKey == nil) {
         status = NO;
-        NSLog(@"\n\n Cannot initialize the SDK. Need to set the API Key. \n\n");
+        [BlueshiftLog logError:nil withDescription:@"Failed to initialize the SDK, API Key is required to initialise the SDK. Set API key in Blueshift config." methodName:nil];
```

Comment:
SDK initialization failed! Please set a valid API key inside the Blueshift config to initialize the SDK.

Reply:
Thanks, will alter the log message.